### PR TITLE
[YUNIKORN-1513] Adding Chinese translation of Development in CodeReady Containers

### PR DESCRIPTION
**Pull request by Paul-Lyu**
### What is this PR for?
Adding Chinese translation of Development in CodeReady Containers

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/YUNIKORN-1513/

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

**Review by 0yukali0 — CHANGES_REQUESTED** [review]
The Chinese translation of documents should be in the https://github.com/apache/yunikorn-site/tree/master/i18n/zh-cn/docusaurus-plugin-content-docs/current.
Because the files in the docusaurus-plugin-content-docs/current follows default language.

**Comment by Paul-Lyu** [comment]
My pull request #309 in wrong place.
But pull request #310 document I put at i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/openshift_development.md.
It is in the docusaurus-plugin-content-docs/current
Could you review again?

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/openshift_development.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+   默认情况下，它应该是 “default-route-openshift-image-registry.apps-crc.testing” 。 如果显示的 URL 不同，请更改上述步骤。
+
+1. 准备 Docker 镜像。
+
+   您可以在底部的 *使用自定义镜像* 部分阅读更多相关信息。
+
+1. 准备 helm chart.
+
+   如果您想使用自定义 Docker 镜像，请替换图表的 “values.yaml” 配置文件中的图像。
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/openshift_development.md:107]: 图表 -> helm chart
图像 -> 镜像

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/openshift_development.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+## 直接推送 OS 镜像注册
+
+1. 创建您要替换的镜像。
+
+   您可以在本地构建新镜像或使用官方镜像（也可以混合使用）。
+      * 对于 -shim 和 -web 镜像检查存储库（可选择进行更改）并输入以下命令：
+      ```bash
+      make clean image REGISTRY=default-route-openshift-image-registry.apps-crc.testing/<project>/<name>:<tag>
+      ```
+      请注意，在 OpenShift 中，一个计划相当于一个 Kubernetes 命名空间。 推荐使用 `yunikorn` 项目/命名空间。
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/openshift_development.md:168]: :::note
在 OpenShift 中，一个计划相当于一个 Kubernetes 命名空间。 推荐使用 `yunikorn` 项目/命名空间。
:::

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/openshift_development.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+1. 启动 CRC 并打开控制台。
+
+   ```bash
+   crc start --pull-secret-file pull-secret.txt
+   crc console
+   ```
+
+## 测试补丁
+
+以下步骤假设您的笔记本电脑中有一个正在运行的 CRC 集群。 请注意，这些步骤未针对远程 CRC 集群进行测试。
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/openshift_development.md:65]: 以下步骤假设您的笔记本电脑中有一个正在运行的 CRC 集群。 
:::note
这些步骤未针对远程 CRC 集群进行测试。
:::

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/openshift_development.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+1. 准备 Docker 镜像。
+
+   您可以在底部的 *使用自定义镜像* 部分阅读更多相关信息。
+
+1. 准备 helm chart.
+
+   如果您想使用自定义 Docker 镜像，请替换图表的 “values.yaml” 配置文件中的图像。
+
+   请注意，如果您直接将 Docker 镜像手动推送到 `default-route-openshift-image-registry.apps-crc.testing` docker registry ，您需要有有效的证书才能访问它。
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/openshift_development.md:109]: :::note
如果您直接将 Docker 镜像手动推送到 `default-route-openshift-image-registry.apps-crc.testing` docker registry ，您需要有有效的证书才能访问它。
在 OpenShift 上有更容易使用的服务：`image-registry.openshift-image-registry.svc`。
:::

**Review by 0yukali0 — CHANGES_REQUESTED** [review]

**Review by 0yukali0 — APPROVED** [review]
LGTM

**Pull request merged**